screens/TabTwoScreen.tsx

```diff
@@ -46,17 +66,24 @@ const App = () =>{
   // }, [])
 
   //search bar
-  const searchFilter = (text : any) => {
-    if (text){
-      const newData = data.filter(function (item){
-        return Object.values(item).some(e => !searchInput || (e as any).toString().toUpperCase().includes(searchInput.toUpperCase()));
-      });
-      setFilterItem(newData);
-      setSearchInput(text);
+  const searchFilter = (text: any) => {
+    if (text) {
+      const newData = data.filter(function (item) {
+        return Object.values(item).some(
+          (e) =>
```

Comment:
Nit: `e` should be reserved as `Event` while here we want to use attributes values on object `item`. Suggest to rename the parameter to something more meaningful to avoid confusion.